cmd/manager/main.go

```diff
@@ -65,7 +66,9 @@ func main() {
 	}
 
 	log.Printf("Initializing Dependencies.")
-	initStaticDeps(mgr)
+	if err := initStaticDeps(mgr); err != nil {
+		log.Fatal(err)
```

Comment:
should this be klog?

Reply:
all logging in main appears to be `log` rather than `klog` currently... 🤔 

@justinsb perhaps we should change this?